response_operations_ui/views/collection_exercise.py

```diff
@@ -1139,20 +1142,43 @@ def view_sample_ci_summary(short_name: str, period: str) -> str:
     return render_template(
         "collection_exercise/view-sample-ci-summary.html",
         collection_instruments=eq_collection_instruments,
+        survey_id=survey_id,
         short_name=short_name,
         period=period,
     )
 
 
-@collection_exercise_bp.route("/<short_name>/<period>/view-sample-ci/summary/<form_type>", methods=["GET"])
+# As a call is already made by 'view_sample_ci_summary' to get the 'survey_id', it didn't make sense to do this call
+# Twice, and so I've included the 'survey_id' as a url param. If we persist 'survey_id' in the future, then this param
+# Be removed
+@collection_exercise_bp.route("/<short_name>/<period>/view-sample-ci/summary/<survey_id>/<form_type>", methods=["GET"])
```

Comment:
I'm torn on this. It does inadvertently open the opportunity for user interference to break or retrieve the wrong set of CIR objects (by hacking the URL). I would personally take the hit on retrieving the survey_id from the short name request. There won't be a huge number of requests being made.